### PR TITLE
gitcoinbot: updated language

### PR DESCRIPTION
**Pull request by thelostone-mc**
##### Description
As discusses on slack,

```
@vivek:  @thelostone-mc related hot-fix request below. 
         Helpful to write up into a ticket or can you handle? (no rush)
         change "Please answer following questions/comments @vs77bb" to something like "@vs77bb,        please see the below comments on approach / questions from the @eswarasai, the bounty hunter."
```

##### Checklist

- [x] linter status: 100% pass
- [ ] changes don't break existing behavior
- [x] commit message follows [commit guidelines](https://github.com/gitcoinco/web/blob/master/docs/CONTRIBUTING.md#step-4-commit)

@mbeacom A lil guidance on the steps to test ? Just filling up the github tokens would suffice ?

### CODE REVIEW

**Comment by thelostone-mc** [comment]
Need to figure out the testing steps but this updates our bot text @vs77bb

File: app/dashboard/notifications.py [hunk 1]
```diff
@@ -402,8 +402,8 @@ def build_github_notification(bounty, event_name, profile_pairs=None):
                     interests = Interest.objects.filter(profile__handle=profile[0], bounty=bounty)
                     for interest in interests:
                         if interest.issue_message.strip():
-                            msg += f"\n__Please answer following questions/comments__ {bounty_owner_clear}:\n\n" + \
-                                    interest.issue_message
+                            msg += f"\n{bounty_owner_clear}, __please see the below comments on the approach / questions from__ " \
```
[the remark this answers is not in the document]
Comment (vs77bb) [reply app/dashboard/notifications.py]: @thelostone-mc This looks good to me. the double tag of the bounty hunter might be overkill but i don't think it's too bad 👍

**Comment by mbeacom** [comment]
@thelostone-mc You just need to setup your local GH secrets and enable notifications for your network (presumably rinkeby).

**Comment by mbeacom** [comment]
@thelostone-mc I just committed the changes to this PR xD

**Pull request merged**